### ISSUE TITLE
Возможно ли получить главы ?

### ISSUE THREAD

**Issue by vladis1av** [NekonexDev/MangaLibAPI#2]

**Comment by DUB1401** [comment]
Да, это возможно. Вы можете воспользоваться, например, моим проектом [MangaLibParser](https://github.com/DUB1401/MangaLibParser), поддерживающим также HentaiLib и YaoiLib. Если немного покопаться в коде, обнаружите все методы получения ссылок на слайды.

Загрузка глав в формате *.CBZ пока тестируется в другом проекте, но уже способна работать вместе с основным форматом JSON моих парсеров. Подробнее смотрите [здесь](https://github.com/DUB1401/RemangaParser?tab=readme-ov-file#%D0%BA%D0%BE%D0%BD%D1%81%D0%BE%D0%BB%D1%8C%D0%BD%D1%8B%D0%B5-%D0%BA%D0%BE%D0%BC%D0%B0%D0%BD%D0%B4%D1%8B).

**Comment by nyakokitsu** [comment]
> _No description provided._

да, мангалиб обновили апи, а я потерял доступ к этому аккаунту

**Comment by nyakokitsu** [comment]
с новым апи в разы все проще
